scripts/convert_yaml_to_sdf.py

```diff
@@ -0,0 +1,207 @@
+                                                            'length': shape_items['cylinder']['height']}}}
+                             })
+            if object_color != [0, 0, 0, 1]:
+                sdf_dict['visual name="{} visual"'.format(shape_name)] = \
+                    {'material': {'ambient': str(object_color).replace("[", "").replace("]", "").replace(",", "")}}
+
+        elif compound_type == "box":
+
+            shape_items_size = [shape_items['box']['size']['x'],
+                                shape_items['box']['size']['y'],
+                                shape_items['box']['size']['z']]
+            shape_items_size = str(shape_items_size).replace("[", "").replace("]", "").replace(",", "")
+
+            sdf_dict.update({'collision name="{}"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'box': {'size': shape_items_size}}},
+                             'visual name="{} visual"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'box': {'size': shape_items_size}}}
+                             })
+            if object_color != [0, 0, 0, 1]:
+                sdf_dict['visual name="{} visual"'.format(shape_name)] = \
+                    {'material': {'ambient': str(object_color).replace("[", "").replace("]", "").replace(",", "")}}
+
+    return sdf_dict
+
+
+def main():
+    # if len(sys.argv) <= 1:
+    #     print "Please provide a file as input"
+    #     sys.exit(1)
+
+    # Create an empty dictionary in SDF format, which can later on be exported to XML after filling below.
+    sdf_dict = {}
+
+    # First determine type of model.yaml
+    # Raw input can be replaced by fancier read_option in create-model.py
+    # file_type = raw_input("File type ('world' or 'object'): ")
+    # model_name = raw_input("Model name: ")
+
+    # Temporary replacements of user input
+    file_type = "object"
+
+    with open("model.yaml", 'r') as stream:
+        try:
+            x = yaml.load(stream)
+            # print(x)
+        except yaml.YAMLError as exc:
+            print(exc)
+    print(x)
+
+    # Filling the empty dictionary
+    if file_type == "object":
+        model_name = 'model name="{}"'.format("couch")
+        # Is object, so create a (static) model type SDF dictionary (as opposed to world), with a default pose
+        sdf_dict = {model_name: {'link name="link"': {},
+                                 'pose': "0 0 0 0 0 0",
+                                 'static': 'true'}}
+        sdf_dict_model_level = sdf_dict[sdf_dict.keys()[0]]
+        sdf_dict_link_level = sdf_dict_model_level['link name="link"']
+
+        if isinstance(x, dict):
+            if 'color' in x:
+                red = x['color']['red']
+                green = x['color']['red']
```

Comment:
I suggest using green for the green color instead of red
```suggestion
                green = x['color']['green']
```